doc/manual/src/expressions/fetchers.md

```diff
@@ -0,0 +1,147 @@
+**Warning**:
+This section is **experimental** and its interface is subject to change.
```

Comment:
How can this be experimental when all the old fetchers are now using this under the hood?

Reply:
I added the warning because of this small detail: https://github.com/NixOS/nix/blob/bceda304982a34c65d5a1dab449cb5bc59f63b83/src/libexpr/primops/fetchTree.cc#L179

Reply:
> How can this be experimental when all the old fetchers are now using this under the hood?

I don’t think here’s a contradiction here: This is indeed used internally, but the interface isn’t stable yet. So it makes sense to mark this as experimental